data/data/powervs/provider.tf

```diff
@@ -1,5 +1,4 @@
 provider "ibm" {
-  version = ">= 1.9.0"
```

Comment:
is this change needed? if yes - I don't see vfsdata is updated with this change! can you run generate code again to get included in the vfsdata?

Reply:
Yes, this is deprecated. Warning is thrown if we have version mentioned.
Hmm yes, I missed generating the code. Will submit a patch.